inception/inception-api-annotation-experimental/src/main/java/de/tudarmstadt/ukp/inception/experimental/api/model/Span.java

```diff
@@ -15,23 +15,29 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package de.tudarmstadt.ukp.inception.experimental.api;
+package de.tudarmstadt.ukp.inception.experimental.api.model;
 
-public class Annotation
+import java.util.List;
+
+public class Span
 {
     private int id;
```

Comment:
How about using the VID here?

Reply:
yes sure, Arcs already had them, forgot to update spans aswell.
Thanks for the hint!